pymetis/__init__.py

```diff
@@ -101,17 +101,19 @@ def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
         unweighted), then the adjwgt can be set to ``None``.
 
     (quoted with slight adaptations from the Metis docs)
+
+    kwargs:
+        *recursive  (bool): Use recursive mode
+        *contiguous (bool): Generate contiguous partitions
+        *ufactor    (int): Specify METIS ufactor
+        *seed       (int): Specify METIS seed
```

Comment:
Why not use conventional keyword arguments rather than fish these out of `**kwargs`?

Reply:
Yeah, I wasn't sure which way to go on this one. In principle there are a lot of METIS options that could
be set, in which case the function signature could get a bit ridiculous. I'm happy to change back to normal keywords if you think it's more appropriate though.

Reply:
(Maybe package the array into a wrapper class?)